### PR TITLE
Account Popover Feature

### PR DESCRIPTION
**Pull request by stevenguyukai**
# Feature
1. Matching colors with this project
2. Shows `Avatar` `Status` `Login time` `Name` `Role` `E-mail` `Description` `Weekly work time` `In process tasks` 

sample

3. Enable Edit for user to adjust their profile
image

# Testing
Styling issue on different devices and different browsers

# Future development
Connect to the data base, retrieve and store personal information

### CODE REVIEW

File: src/components/AccountPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        fourthWeek: 6,
+        fifthWeek: 10,
+        sixthWeek: 8,
+        seventhWeek: 7,
+        eighthWeek: 10,
+        ninthWeek: 7,
+        tenthWeek: 8
+    },
+    tags: ["Developer", "Volunteer"],
+    tasks: ["Task 1: Finish the personal information page", "Task 2: Looking into backend APIs", "Task 3: Implement the user profile"],
```
Comment (jessieh9) [inline src/components/AccountPopup.jsx]: Don't need these tasks right now. Check the description + Figma.

File: src/components/AccountPopup.jsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                                <ResponsiveContainer width="100%" height={60}>
+                                    <BarChart data={data}>
+                                        <RechartsTooltip content={<CustomTooltip />} />
+                                        <Bar dataKey="hours" fill="#4498ec" />
+                                    </BarChart>
+                                </ResponsiveContainer>
+                            </VStack>
+                            <Divider/>
+                            <VStack align="left">
+                                {userData.tasks.map((task, index) => (
```
Comment (jessieh9) [inline src/components/AccountPopup.jsx]: same thing, don't need the tasks for now :)

File: src/components/AccountPopup.jsx [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    const navigate = useNavigate();
+    const supabase = useSupabaseClient();
+
+    async function signOut() {
+        await supabase.auth.signOut().then(() => {
+            navigate("/Login");
+        });
+    }
+
+    const data = Object.entries(userData.hours).map(([week, hours]) => ({
```
Comment (jessieh9) [inline src/components/AccountPopup.jsx:76]: I think the Weeks all show Week 1 in the demo when I ran your code. Can you edit to enable showing Week 1-10? Thanks.

**Review by jessieh9 — CHANGES_REQUESTED** [review]

**Comment by stevenguyukai** [comment]
I have commanding out `task`, and fix the chart that always display "week 1". Also I don't have the link to the Figma :o

**Comment by jessieh9** [comment]
> I have commanding out `task`, and fix the chart that always display "week 1". Also I don't have the link to the Figma :o

Oh hm, for Figma link is in the frontend chat of our discord but also here for easy access https://www.figma.com/team_invite/redeem/m7USct7pyXUSdm1R6BbS9B

**Review by jessieh9 — APPROVED** [review]
LGTM, thanks for the changes.

**Pull request merged**